Monika After Story/game/script-topics.rpy

```diff
@@ -11514,8 +11538,43 @@ label monika_grad_speech_call:
     return
 
 label monika_grad_speech_not_paying_attention:
-    #First menu timeout
     hide screen mas_background_timed_jump
+
+    # Make a double check here.
+    m 1euc "...[player]?{w=0.3}{nw}"
+    m 1wud "Are you...{w=0.3}listening?{nw}"
+    $ _history_list.pop()
+    show screen mas_background_timed_jump(5, "monika_grad_speech_not_paying_attention_real")
+    menu:
+        m "Are you...listening?{fast}"
+        "Of course!":
+            hide screen mas_background_timed_jump
+
+            m 1hub "Ah, sorry for assuming~"
+            m 1hua "Anyways, what do you think?{nw}"
+        
+        "Of course not!":
```

Comment:
Not sure someone would actually say it that way, I think it could be phrased other way, not so mean.

Reply:
Ah, I think this is a joke.Like online chat:

A: "Are you there?"
B: "No."

B was obviously there -- or how could he have replied?

Reply:
I'd change to `"What?"`

Reply:
Actually, I think in the original idea @Legendkiller21 wanted to show just one option, so we can remove this one.

Reply:
Well, [farewell, my menu option](https://github.com/Monika-After-Story/MonikaModDev/pull/9644/commits/465493bfc35842579cdd99b8913c1169eaaaf7c9).